app/src/main/java/com/bdb/bikedeboa/util/BindingAdapters.java

```diff
@@ -28,7 +28,7 @@ public static void loadImage(ImageView imageView, String imageAddress) {
 				.thumbnail(Glide.with(imageView.getContext())
 						.load(R.drawable.wheel_loading))
 				.crossFade()
-				.diskCacheStrategy(DiskCacheStrategy.RESULT)
+				.diskCacheStrategy(DiskCacheStrategy.SOURCE)
```

Comment:
Result pega a imagem com tamanho original da internet, escala ela pra caber na imageview, e salva a imagem escalada no cache. O que ajuda a gastar menos memória.

O Source pega a imagem original da internet, e salva a imagem como ela vem. Se ela é 1000x1000 px e é vista numa imageview de 100x100 px, ele cacheia a imagem 1000x1000 na memória (enquanto o RESULT salvaria a 100x100).

Eu mudei pra Source pq o que tava acontecendo com Result era:
- Eu baixava a imagem uma vez pra view de detalhes.
- Daí a lib cacheava a imagem escalada.
- Quando eu clicava na imagem pra expandir ela, o app tinha que baixar a imagem uma segunda vez, pq a imagem cacheada era pequena pra view de expansão. O que torna o cache meio inútil. 
-> Usando a estratégia de Source, eu evito fazer esse segundo download. Ele pega a imagem original da internet, e pras duas views usa a mesma.